pootle/apps/pootle_store/models.py

```diff
@@ -1864,6 +1877,20 @@ def insert(i1, j1, j2, offset):
                     self.get_max_unit_revision())
                 )
 
+    def increment_unsynced_unit_revision(self, update_revision):
+        filter_by = {
+            'store': self,
+            'revision__gt': self.last_sync_revision,
+            'revision__lt': update_revision,
+        }
+        count = 0
+        for unit in Unit.simple_objects.filter(**filter_by) \
+                        .exclude(state=OBSOLETE):
```

Comment:
Ok, but using the `filter_by` I proposed should still make it a bit more readable.